README.md

```diff
@@ -252,7 +252,7 @@ module "lb" {
 | load\_balancer\_type | The type of load balancer to create. Possible values are application or network. | `string` | `"application"` | no |
 | load\_balancer\_update\_timeout | Timeout value when updating the ALB. | `string` | `"10m"` | no |
 | name | The resource name and Name tag of the load balancer. | `string` | `null` | no |
-| name\_prefix | The resource name prefix and Name tag of the load balancer. | `string` | `null` | no |
+| name\_prefix | The resource name prefix and Name tag of the load balancer. Cannot be longer than 6 characters. | `string` | `null` | no |
```

Comment:
Please update the content of `variables.tf` where this description of variables is taken from by `pre-commit-terraform` hook.

Reply:
Thanks for the correction I didn't noticed that. I've commited a new change with the observation.